apps/desktop/src/app/app-routing.module.ts

```diff
@@ -331,6 +332,10 @@ const routes: Routes = [
       children: [{ path: "", component: LoginDecryptionOptionsComponent }],
     },
   ),
+  {
+    path: "passkeys",
```

Comment:
nit: just to be consistent with naming scheme https://contributing.bitwarden.com/architecture/deep-dives/passkeys/naming-convention
```suggestion
    path: "fido2",
```